newIDE/app/src/AssetStore/ExtensionStore/index.js

```diff
@@ -117,6 +119,7 @@ export const ExtensionStore = ({
             selectedExtensionShortHeader.name
           )}
           onInstall={() => {
+            onInstallExtension(selectedExtensionShortHeader);
```

Comment:
There is already a prop doing this, so we don't need this duplicate. Let's use the other in the parent :)